README-Automation.md

```diff
@@ -0,0 +1,195 @@
+#Automation Scripts
+
+## Table of Contents
+
+- [Services](#services)
+- [Installing The Automation Scripts](#installing-the-automation-scripts)
+  - [Stable Channel](#stable-channel)
+  - [Unstable Channel](#unstable-channel)
+- [Initializing Drive](#initializing-drive)
+- [Using The Automation Scripts](#using-the-automation-scripts)
+  - [Setting Up](#setting-up)
+  - [Selecting Objects](#selecting-objects)
+  - [Launching A Script](#launching-a-script)
+  - [Help](#help)
+- [drive-menu Screenshots](#drive-menu-screenshots)
+- [Tracking And Filing Issues](#tracking-and-filing-issues)
+- [Reaching Out](#reaching-out)
+- [Disclaimer](#disclaimer)
+- [LICENSE](#license)
+
+## Services
+
+JC Manciot's [PPA](https://gitlab.com/jean-christophe-manciot/ppa) offers a collection of scripts for automated drive commands including syncing which can be used on the latest stable Ubuntu distribution.
```

Comment:
The phrasing is **very clear** and relates to my PPA philosophy. ~ Every six months, all packages are rebuilt for the new Ubuntu version. After a short period to let people upgrade, all previous builds are removed. You seem to have a good understanding of Ubuntu releases cycle, but let me state the difference between LTS and stable:
- **[LTS](https://wiki.ubuntu.com/Releases)** means "Long Term Support". 
- **Stable** is opposed to unstable: 16.10 and earlier non-EoL versions are considered stable, 17.04 is currently developed and considered unstable. 

So stable and LTS are 2 different concepts.
I **cannot issue a PR each semester** for this file, hence the generic 'latest stable...".
If you take a look at the package dependencies and the way the scripts are written (if you're familiar with bash), you'll see that there is a high chance that they work with earlier Ubuntu versions; but, they **won't be tested within those**. So, it's up to any user of earlier Ubuntu releases to manually download and install "drive-google" .package from my [Drive static repository](https://gitlab.com/jean-christophe-manciot/Drive) and see for him/herself whether it suits his/her needs ... or not.

I won't test all use cases, only on the last stable Ubuntu version.
However, if you discover some issues with earlier non-EoL Ubuntu versions, you can post them and I'll take a look at them.

I have nevertheless added the mention "latest stable Ubuntu release" beside both channels to clarify the situation.